UnicornRender/video/include/unicorn/video/Renderer.hpp

```diff
@@ -68,30 +69,29 @@ class Renderer
      */
     UNICORN_EXPORT virtual void SetDepthTest(bool enabled) = 0;
 
-    /** @brief  Creates new geometry mesh
-    *
-    *  Creates and subscribes to mesh.
-    *  Mesh shall be deleted via DeleteMesh().
-    *
-    *  @param [in] material describes mesh visual representation
-    *
-    *  @attention  Mesh lifetime is bound by its renderer's lifetime.
-    *              Using meshes after their renderer was destroyed is undefined behaviour.
-    *              If you're storing mesh pointers, consider storing them with a reference
-    *              to their renderer and listening for its Destroyed event for proper cleanup.
-    *
-    *  @return pointer to newly created geometry::Mesh
-    *
-    *  @sa DeleteMesh
+    /**
+    * @brief Adds mesh to rendering system
+    * @param [in] mesh mesh data
+    */
+    UNICORN_EXPORT virtual void AddMesh(Mesh* mesh) = 0;
+
+    /**
+    * @brief Adds multiple meshes simultaneously to rendering system
+    * @param [in] meshes meshes data
     */
-    UNICORN_EXPORT virtual Mesh* SpawnMesh(Material const& material) = 0;
+    UNICORN_EXPORT virtual void AddMeshes(std::list<Mesh*> const& meshes) = 0;
 
     /**
     * @brief Removes mesh from rendering system
-    * @param [in] mesh pointer to mesh
-    * @return true if mesh was sucessfully deleted from system and false if not
+    * @param [in] pMesh pointer to mesh
+    */
+    UNICORN_EXPORT virtual bool DeleteMesh(Mesh const* pMesh) = 0;
+
+    /**
+    * @brief Removes multiple meshes simultaneously from rendering system
+    * @param [in] meshes list of pointer to meshes
```

Comment:
where's `@return`? I don't think this bool is needed at all - what if you only deleted _some_ of the meshes, but not all - you can't indicate which. I don't think this method is useful, it doesn't let user know which pointers are valid and which are not

Reply:
The idea of deleting function is just delete from rendering system. Not from from ram. Pointer is used as "search id".